scripts/actions/mobskills/synergism.lua

```diff
@@ -0,0 +1,35 @@
+-----------------------------------
+-- Synergism
+-- Description: Steals HP from any nearby flans.
+-- Type: Magical
+-----------------------------------
+local mobskillObject = {}
+
+mobskillObject.onMobSkillCheck = function(target, mob, skill)
+    return 0
+end
+
+mobskillObject.onMobWeaponSkill = function(target, mob, skill)
+    if target:getID() == mob:getID() then
+        local absorbed = 0
+        local count = skill:getTotalTargets() - 1  -- Do not include self
+        if count > 0 then
+            local percentage = math.max(0.3, (100 - skill:getMobHPP()) / 200)
```

Comment:
Is there a reason for the math.max? Wouldnt this mean anything under 30% gets defaulted to 30%

Reply:
You're right, this doesn't belong. It was something I was experimenting with while trying to figure out the algorithm. I removed it.